en/docs/deploy/configure-email-sending.md

```diff
@@ -46,5 +46,116 @@ This document explains the steps to configure WSO2 Identity Server to send email
 
 3. Save the configurations and start the server.
 
+## Configure the email sender (per tenant)
+
+Follow the steps given below to enable the email sender per tenant.
+
+1.  Configure the [Configuration Management REST API]({{base_path}}/apis/use-the-configuration-management-rest-apis/). 
+2.  Execute the following curl command to create a resource type named `Publisher`. 
+
+    **Sample Request**
+    ``` java 
+    curl -X POST "https://localhost:9443/t/{tenant-domain}/api/identity/config-mgt/v1.0/resource-type" -H "accept: 
+    application/json" -H 
+    "Content-Type: application/json" -H 'Authorization: Basic YWRtaW46YWRtaW4=' -d "{ \"name\": \"Publisher\", \"description\": \"Publisher Configurations\"}"
+    ```
+
+3.  Execute the following curl command for creating a resource named `EmailPublisher`. 
+
+    **Sample Request**
+    ``` java 
+    curl -X POST "https://localhost:9443/t/{tenant-domain}/api/identity/config-mgt/v1.0/resource/Publisher" -H "accept: 
+    application/json" -H "Content-Type: application/json" -H 'Authorization: Basic YWRtaW46YWRtaW4=' -d "{ \"name\": \"EmailPublisher\", \"attributes\": [ { \"key\": \"email\", \"value\": \"string\" } ]}"
+    ```
+	
+4.  Execute the following curl command for creating a file named `EmailPublisher`. 
```

Comment:
Can we differentiate between point 3 and point 4? They have the same instructions.

Reply:
Noted. Need to test this.
Shall we fix these during the testing effort tracked by the excel? Only did the refactoring here.